js/tabs.js

```diff
@@ -25,7 +25,7 @@
     
     if (response.activeTab) {
       $('.contacts-ajax-tabs .' + activeClass).removeClass(activeClass);
-      $('.contacts-ajax-tabs .' + response.activeTab).find('a').andSelf().addClass(activeClass);
+      $('.contacts-ajax-tabs .' + response.activeTab).find('a').addBack().addClass(activeClass);
```

Comment:
This is presumably also for an update to jquery with a Drupal update? Could go with the ajax.js change?